components/layout/sequential.rs

```diff
@@ -78,7 +78,6 @@ pub fn build_display_list_for_subtree(flow_root: &mut Flow,
                                       root_stacking_context: &mut StackingContext,
                                       shared_layout_context: &SharedLayoutContext)
                                       -> Vec<DisplayItem> {
-    flow_root.traverse_preorder(&ComputeAbsolutePositions { layout_context: shared_layout_context });
```

Comment:
Well, it did at one point, but I removed it because it was slower than sequential on all the Web sites I tried. Right now, this pass is always done sequentially.